tools/fixedpointarithmetic/src/main/java/dk/alexandra/fresco/decimal/RealNumeric.java

```diff
@@ -0,0 +1,26 @@
+package dk.alexandra.fresco.decimal;
+
+public interface RealNumeric {
```

Comment:
I think this naming is a little confusing. I would expect RealNumeric to be a counterpart to Numeric, but that does not seem to be the case.